rails/app/models/medium.rb

```diff
@@ -0,0 +1,42 @@
+class Medium < ApplicationRecord
```

Comment:
This is a super pedantic nit that you can completely ignore, but any thoughts on configuring rails inflections so we can name the singular model `Media` as well? In a few places when you loop over multiple `media` instances, you call the block argument `media` as well so it seems like you agree on the singular form in this context. Coincidentally [this stackoverflow post covers custom inflections](https://stackoverflow.com/questions/39585774/pluralizations-and-singularizations-inflections-for-media-medium-in-rails-5) with this exact same use case, ignoring the scaffold part. 

To stay on the pedantic tone, I believe we're using this specific meaning which does have a singular form of "media" https://en.wiktionary.org/wiki/media#Noun_3

Reply:
Yeah, I waffled on it. Like for a while. You may have noticed the underlying table is “media”

@rudokemper do you have a preference?

Reply:
By the way - love the pedantic. Keep it up!

Reply:
Same!

I'd opt for `Media` in singular.

Reply:
Well, turns out we can have the best of both by just specifying the irregular singularity of media is media.